incentivizing_rewarding_participants.adoc

```diff
@@ -1,87 +1,157 @@
-
-Lorem ipsum dolor sit amet, consectetur adipiscing elit.
-Suspendisse viverra id leo et lacinia.
-Maecenas convallis lacinia diam, et lobortis quam tincidunt hendrerit.
-Quisque nisl eros, hendrerit et erat sit amet, feugiat venenatis sapien.
-Duis a dapibus mauris.
-
-=== Karma and badge systems
+Open source community managers and architects strive to create places where participants feel valued for the contributions they make.
+When contributors feel as though their work matters, they're more likely to _continue_ contributing to a project, enhancing the community and impacting its culture as they do.
+Contributors derive the most value from participating in an open source project when the project satisfies or otherwise _rewards_ their motivations for contributing in the first place.
+For community architects, the crux of the challenge in creating rewarding contributor experiences is aligning participant motivations with various models and mechanisms for rewarding project contributions.
+
+But not everyone contributres to open source projects for the same reasons.
+For example, https://arxiv.org/abs/2101.10291[recent research on motivations] for contributing to open source projects suggests that altruism and kinship-seeking impact contributor motivations more significantly than they did a decade ago.
+Moreover, participants report that their motivations for _continuing_ contributions to a project aren't always the same as their motivations for _initially contributing_ to a project.
+Someone may join a project in order to contribute a patch her employer wanted to see adopted upstream, but remain in the project after identifying project participants as a peer group and developing a sense of kinship with members.
+Building open source communities involves responding to contributors' continually shifting motivations for participating with new activities and opportunities from which they might derive value, incentivizing them to continue contributing.
+
+Not every contributor to your open source project will develop a deep level of investment in the project.
+You won't always be able to convert infrequent, casual contributors into consistent, repeat contributors—nor should that necessarily be your goal.
+More important is understanding how participants' contributions align with or reflect their motivations for _making_ those contributions—and responding accordingly.
+As https://opensource.com/article/17/10/managing-casual-contributors[researcher Ann Barcomb writes], "People who contribute to boost their CV or to fix an irritating bug are less likely to continue once their need is satisfied."
+Alternatively, however, these periodic contributors (or what Barcomb calls "episodic contributors") often seek low-complexity, easily explainable, and quickly addressable issues—the very kinds of issues long-time contributors would love to see others solve so they can focus on other matters.
```

Comment:
There could be a paragraph break about here, with the goal of making it easier to read. Does that seem doable here in the middle?